_posts/2024-08-22-announcing-ballerina-2201.10.0-swan-lake-update-10.md

```diff
@@ -0,0 +1,51 @@
+
+## OpenAPI Tool
+
+A major highlight of Ballerina Swan Lake Update 10 is the introduction of extensive enhancements to the OpenAPI tool. 
+A contract-first approach is introduced as a standout key feature using the OpenAPI tool and the HTTP package. By generating a service contract type directly from the OpenAPI specification, you can design services that are contract-compliant from the outset. In addition, centralizing metadata within the service contract type results in a cleaner, more maintainable codebase. 
+
+<img alt="Swagger UI" src="/images/swagger_ui.png">
+
+Another important feature is the addition of example annotations on Ballerina types, resource parameters, and record fields, which are now directly rendered into the OpenAPI example schema. This provides developers with clear, contextually relevant examples, making APIs easier to understand and implement. For more information, check this [link](https://ballerina.io/learn/openapi-tool/#export-with-given-examples-information). 
+
+To further streamline development, the update introduces mock client generation with examples directly from the OpenAPI specification. Picture a developer tasked with integrating a new API: you can now effortlessly generate mock clients with examples that you documented in the OAS, test and validate integrations, and then seamlessly replace the mock client with the actual one when ready for production. For more information, check this [link](https://ballerina.io/learn/openapi-tool/#generate-mock-client-using-included-example-in-oas).
+
+The OpenAPI package allows developers to automatically generate and embed the OpenAPI specification for a Ballerina service via a resource. This has been improved by exposing another resource to view the HTML-rendered version of the OpenAPI specification. This feature allows you to introspect the service through a more intuitive and readable format. For more information, see [OpenAPI Specification resources](https://ballerina.io/spec/http/#236-openapi-specification-resources)
+
+<img alt="Service contract type" src="/images/service_contract_type.png">
+
+## New Connectors
+
+In this release, we’ve made substantial improvements to the functionality and usability of our connectors. Here are the key highlights:
+As part of the connector revamp initiative, several connectors have been significantly updated. The newly released `dayforce` and `discord` packages now support interactions with their latest REST API versions. Additionally, `asb`, `slack`, `stripe`, and `twitter` (X) connectors have received major updates, featuring significant API and functionality changes, along with enhanced documentation, making it easier to develop complex applications.
```

Comment:
Don't these have to be indented? Have we checked how all of these get rendered?

Reply:
Added bullets